uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -841,19 +593,14 @@ define([
      *
      * @param {String} dateType from|to
      */
-    function resetUIDayTypesTimeAndDeductions (dateType) {
+    function resetUIInputs (dateType) {
```

Comment:
`resetUIDayTypesTimeAndDeductions` was more specific, why rename it?

Reply:
It is now a generic function, amended it, there was a mistake actually.